os/include/tinyara/pm/pm.h

```diff
@@ -350,9 +270,22 @@ struct pm_callback_s {
 	 *
 	 **************************************************************************/
 
-	void (*notify)(FAR struct pm_callback_s *cb, int domain, enum pm_state_e pmstate);
+	void (*notify)(FAR struct pm_callback_s *cb, int domain, int pmstate);
 };
 
+#if defined(CONFIG_PM) && !defined(CONFIG_PM_NSTATE)
+#error "SoC specific PM not enabled, for example CONFIG_S5J_PM is not set"
```

Comment:
```for example CONFIG_S5J_PM is not set"```
> You check only CONFIG_PM_NSTATE in conditional but you mention CONFIG_S5J_PM.

Reply:
Because once PM is enabled, SOC has to provide CONFIG_PM_NSTATE value
& because we don't add  architecture specific check in kernel code, have used CONFIG_PM_NSTATE check.

Reply:
I mean ```for example CONFIG_S5J_PM is not set``` can give confusion.
Why don't you mention CONFIG_PM_NSTATE even though you check it in conditional?

Reply:
ok will mention it